docs/usage/templates.md

```diff
@@ -85,3 +85,11 @@ In the example above, it will only show a text if `isMajor=true` and `hasRelease
 Returns `true` if at least one expression is `true`.
 
 `{{#if (or isPatch isSingleVersion}}Small update, safer to merge and release.{{else}}Check out the changelog for all versions before merging!{{/if}}`
+
+## Environment variables
+
+For security reasons, you can only access a handful of basic environment variables like `HOME` or `PATH` by default:
```

Comment:
We should mention `exposeAllEnv`[^1] here to allow all env. @HonkingGoose Can you help?

[^1]: https://docs.renovatebot.com/self-hosted-configuration/#exposeallenv